### ISSUE TITLE
Class methods mustn't be converted to properties

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#84]
Thanks for fixing #81 - there is a small bug, however. If the method is a class method (i.e. it's invoked directly on the class), it musnt' be converted to a property (it must stay a method).

Example:

```
[NSString defaultCStringEncoding];
```

Currently translated to:

```
NSString. defaultCStringEncoding
```

Should be:

```
NSString. defaultCStringEncoding()
```

**Comment by drkameleon** [comment]
Hmm... Good catch :) Noted.

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 2.0.
